### PR TITLE
feature(docs): add glossary to Falco website

### LINKED ISSUE #1006: Create a glossary for Falco terms
/area documentation

**What would you like to be added**:
- Improve understanding of Falco content - should be understandable by someone that doesn’t know C++, anything about the kernel
- Provide references for individuals to learn more about eBFP, kernel security, syscalls, and more.

**Why is this needed**:
users need a good context about Falco terms.

```[tasklist]
### Tasks
- [ ] Define and add references for syscalls
- [ ] Define and add references for Kernel
- [ ] Kernel space and user space
- [ ] Define and add references for alerts
- [ ] Define and add references for gRPC
- [ ] Define and add references for Runtime Security
- [ ] Define and add references for plugins
- [ ] Define and add references for extractors
- [ ] Define and add references for sources
- [ ] Define and add references for drivers
- [ ] Define and add references for eBPF
- [ ] Define and add references for probe
- [ ] Define and add references for module
- [ ] Define and add references for rule
- [ ] Define and add references for rules file
- [ ] Define and add references for macro
- [ ] Define and add references for condition
- [ ] Define and add references for  list
- [ ] Define and add references for output
- [ ] Define and add references for rule fields
- [ ] Define and add references for ring buffer
- [ ] Define and add references for exporter
- [ ] Define and add references for event generator
- [ ] Define and add references for priority
- [ ] Define and add references for tags
- [ ] Define and add references for exceptions
```

### PR DESCRIPTION
**Pull request by vjjmiras**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

/kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

/area documentation

> /area videos

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1006 

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/docs/reference/glossary/syscalls.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Syscalls
+id: syscalls
+date: 2023-07-17
+full_link: https://man7.org/linux/man-pages/man2/syscalls.2.html
+short_description: >
+  Syscalls are for system calls, the way to communicate with the kernel.
```
Comment (FedeDP) [inline content/en/docs/reference/glossary/syscalls.md:7]:
```suggestion
  Syscalls stands for system calls, a way to request a service from the running kernel.
```

File: content/en/docs/reference/glossary/syscalls.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+aka:
+tags:
+- fundamental
+---
+Syscalls are for system calls, the way to communicate with the {{< glossary_tooltip text="kernel" term_id="kernel" >}}.
```
Comment (FedeDP) [inline content/en/docs/reference/glossary/syscalls.md:12]:
```suggestion
Syscalls stands for system calls, a way to request a service from the running {{< glossary_tooltip text="kernel" term_id="kernel" >}}.
```

File: content/en/docs/reference/glossary/syscalls.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: Syscalls
+id: syscalls
+date: 2023-07-17
+full_link: https://man7.org/linux/man-pages/man2/syscalls.2.html
+short_description: >
+  Syscalls are for system calls, the way to communicate with the kernel.
+aka:
+tags:
+- fundamental
```
Comment (FedeDP) [inline content/en/docs/reference/glossary/syscalls.md:10]:
```suggestion
- fundamental
- event source
```

**Review by FedeDP — COMMENTED** [review]
(I see that the correct tag is `event-source` btw)

**Review by leogr — DISMISSED** [review]

File: content/en/docs/reference/glossary/falcosidekick-ui.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+id: falcosidekick-ui
+date: 2023-07-17
+full_link: https://github.com/falcosecurity/falcosidekick-ui
+short_description: >
+  A simple WebUI for displaying the latest events from Falco, it works as an output for Falcosidekick.
+aka:
+tags:
+- integration
+---
+A simple WebUI for displaying the latest events from {{< glossary_tooltip text="Falco" term_id="falco" >}}, it works as an output for {{< glossary_tooltip text="Falcosidekick" term_id="falcosidekick" >}}.
```
[the remark this answers is not in the document]
Comment (vjjmiras) [reply content/en/docs/reference/glossary/falcosidekick-ui.md:12]: It is consistent with https://github.com/falcosecurity/falcosidekick-ui

**Review by leogr — APPROVED** [review]

**Pull request merged**